gcc/rust/gccrs.texi

```diff
@@ -0,0 +1,207 @@
+\input texinfo @c -*-texinfo-*-
+@setfilename gccrs.info
+@settitle The GNU Rust Compiler
```

Comment:
Just to be sure, is that the "blessed" name, wrt the Rust® ?

Reply:
good question :) I think we can leave it like this for now, and be open to changing it